plugins/Makefile

```diff
@@ -2199,6 +2207,13 @@ $(BUILD_DIR)/cf/src/%.cpp.o: cf/src/%.cpp
 		-DpluginInstance=pluginInstance__cf \
 		-Wno-misleading-indentation
 
+$(BUILD_DIR)/CVfunk/src/%.cpp.o: CVfunk/src/%.cpp
+	-@mkdir -p "$(shell dirname $(BUILD_DIR)/$<)"
+	@echo "Compiling $<"
+	$(SILENT)$(CXX) $< $(BUILD_CXX_FLAGS) -c -o $@ \
+		$(foreach m,$(CF_CUSTOM),$(call custom_module_names,$(m),cf)) \
```

Comment:
this needs to be `CVFUNK_CUSTOM` instead of `CF_CUSTOM`. also the `cf` later should match the makefile side so make it exactly `CVfunk`